share/input_wrf.F

```diff
@@ -202,7 +202,7 @@ SUBROUTINE input_wrf ( fid , grid , config_flags , switch , ierr )
           CALL wrf_get_dom_ti_integer( fid , 'HYBRID_OPT',  hybrid_opt,  1, icnt, ierr )
           CALL wrf_get_dom_ti_integer( fid , 'USE_THETA_M', use_theta_m, 1, icnt, ierr )
 
-          IF ( hybrid_opt .NE. config_flags%hybrid_opt ) THEN
+          IF ( hybrid_opt .NE. config_flags%hybrid_opt .and. hybrid_opt .le. 100 ) THEN
```

Comment:
@children1985 
Can you explain what the extra test `hybrid_opt .le. 100` is for? I assume that you are using the same logic for the moist theta (`use_theta_m`).

Reply:
This has been addressed